jobdispatcher/src/main/java/com/firebase/jobdispatcher/JobTrigger.java

```diff
@@ -16,9 +16,10 @@
 
 package com.firebase.jobdispatcher;
 
-/* package */ class JobTrigger {
-    /* package */ static final class ImmediateTrigger extends JobTrigger {
-        /* package */ ImmediateTrigger() {}
+public class JobTrigger {
+
+    public static final class ImmediateTrigger extends JobTrigger {
+        public ImmediateTrigger() {}
```

Comment:
please keep the constructor package private (w/ the `/* package */` comment intact)